bridge_adaptivity/module/static/module/js/forbid_next.js

```diff
@@ -1,54 +1,65 @@
-(function(){
-    var forbidNext = $("#next-button").data("forbidden");
-    if (forbidNext){
-        console.log("Current Activity is not answered, NEXT is forbidden!");
-        $("#forbidNextModal").modal("show");
-    }
-}());
+// include `module/js/work_with_cookie.js` in template before using this script
 
-var enable_next_buttons = function () {
+function enable_next_buttons() {
     $(".next-button-link button.disabled").removeClass("disabled");
     next_item = $("#next-button").data("next_item");
     $(".next-button-link").attr("href", next_item)
-};
-
-var update_ui_details = function(uiDetailsArray){
+}
 
+function update_ui_details(uiDetailsArray) {
     ui_option_element = $("#ui_option");
     child_elements = ui_option_element.data("child_elements");
     child_classes = ui_option_element.data("child_classes");
     ui_option_element.empty();
-    uiDetailsArray.forEach(function(ui_option){
-        var new_element = document.createElement(child_elements);
+    uiDetailsArray.forEach(function(ui_option) {
+        let new_element = document.createElement(child_elements);
         new_element.className = child_classes;
         new_element.innerHTML = ui_option;
         ui_option_element.append(new_element);
     })
-};
+}
+
+function show_congratulation_pop_up(congratulation_cookie_name){
```

Comment:
Why did you decide to use cookies to show the message

Reply:
It needs for showing this message only once time if the user has enough score to do that.

Reply:
I could be cached on the backend doesn't it

Reply:
We talked about it. Cookie works well with it.  In my point, this is a frontend's work to show the message or not.